task/sast-snyk-check/0.3/sast-snyk-check.yaml

```diff
@@ -46,11 +46,7 @@ spec:
       default: "true"
     - name: KFP_GIT_URL
       type: string
-      description: Known False Positives (KFP) git URL (optionally taking
-        a revision delimited by \#). Defaults to "SITE_DEFAULT", which means
-        the default value "https://gitlab.cee.redhat.com/osh/known-false-positives.git" for internal Konflux
-        instance and empty string for external Konflux instance.
-        If set to an empty string, the KFP filtering is disabled.
+      description: Known False Positives (KFP) git URL (optionally taking a revision delimited by \#). Defaults to "SITE_DEFAULT", which means the default value "https://gitlab.cee.redhat.com/osh/known-false-positives.git" for internal Konflux instance and empty string for external Konflux instance. If set to an empty string, the KFP filtering is disabled.
```

Comment:
This change was not needed and introduces an unnecessary difference between the 0.3 and 0.4 versions of the task.